compose-builder/README.md

```diff
@@ -55,8 +55,24 @@ This folder contains the following compose files:
     Device Service **extending** compose file, which adds the **Device SNMP**  service.
 - **add-device-virtual.yml**
     Device Service **extending** compose file, which adds the **Device Virtual**  service.
+- **add-asc-http-export.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service Http Export**  service.
+- **add-asc-http-export-secure.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service Http Export Secrets** service. Service is enable for secure secrets.
+- **add-asc-mqtt-export.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service MQTT Export**  service.
+- **add-asc-MQTT-export-secure.yml**
+    Application Service Configurable **extending** compose file, which adds the **App Service Http Export Secrets** service. Service is enable for secure secrets.
```

Comment:
Should **App Service Http Export Secrets** be **App Service MQTT Export Secrets**?

Reply:
Yes! good catch. Thanks!

Reply:
fixed